src/App.tsx

```diff
@@ -15,71 +15,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type AppState = {
+  selectedItem: string,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, AppState> {
+  state = {
+    selectedItem: goods[goods.indexOf('Jam')],
```

Comment:
Yeah, it's not needed here, use 'Jam', or if you want first element just do goods[0]